src/platform/datapath_raw_socket.c

```diff
@@ -118,14 +118,24 @@ CxPlatDpRawParseUdp(
         return;
     }
 
+    if (Length < QuicNetByteSwapShort(Udp->Length)) {
```

Comment:
nit: maybe we should store the UDP length from UDP header in a local var so we don't have to do the swap twice. Though, that swap might be completely optimized out.

Reply:
I'll leave it as is for now. Let's see if the compiler can deal with it.